### DISCUSSION TITLE
Popups

### DISCUSSION THREAD

**Discussion by Midnayt** [nalu-development/nalu#51] [General]
Hi, does your lib supports any popups(CommunityPopup/Mopups)?

**Comment by albyrock87** [comment]
@Midnayt as far as I can tell, any popup - like `CommunityToolkit/Popup` or `CommunityPopup/Mopups` or `DevExpress/DXPopup` - does not rely on MAUI navigation system to display the popup.

In general a popup is not part of the navigation system but it's something shown on top of the current page.

So, yes, my library should be compatible due to the fact we're simply talking about different things.
```cs
// Use INavigationService to push a new page
_navigationService.GoToAsync(Navigation.Relative().Push<DetailPageModel>());
// Use Mopup to display a popup
_mopupService.PushAsync(new MyPopupPage { Animation = animation });
```

If you find yourself in a position where `Mopups` is not working using my library feel free to open an issue and I'll take a look as quickly as I can.

**Comment by Midnayt** [comment]
@albyrock87 Thank you for your response. Will check and then report if there will be an issues.
any specific registration for popups?

**Reply by albyrock87** [reply]
I think you should refer to the popup library docs you're using.
From this library point of view you don't need anything specific.

**Comment by albyrock87** [comment]
@Midnayt I just release Nalu Popups within 9.7.0 release.

See details here: https://nalu-development.github.io/nalu/layouts.html#popups

I've also updated the support for the new version of Community Toolkit popups, see #88.
